Dockerfile

```diff
@@ -1,16 +1,46 @@
-FROM 		phusion/baseimage
-MAINTAINER 	Casey Bisson <casey.bisson@gmail.com
+FROM alpine:3.2
+MAINTAINER 	Casey Bisson <casey.bisson@gmail.com>
 
-# Update Apt
-RUN apt-get update
+# Alpine packages
+RUN apk --update \
+    add \
+        jq \
+        curl \
+        bash \
+        ca-certificates && \
+    curl -Ls https://circle-artifacts.com/gh/andyshinn/alpine-pkg-glibc/6/artifacts/0/home/ubuntu/alpine-pkg-glibc/packages/x86_64/glibc-2.21-r2.apk > /tmp/glibc-2.21-r2.apk && \
```

Comment:
Interesting. You curl in glibc. Wouldn't it be better to build Consul using Musl?

```
# Download and install Consul
# We are building it ourselves, because Alpine Linux doesn't use glibc
RUN export GOPATH=/go && \
    apk add --update go git gcc musl-dev make bash && \
    go get github.com/hashicorp/consul && \
    cd $GOPATH/src/github.com/hashicorp/consul && \
    git checkout -q --detach "v$CONSUL_VERSION" && \
    make && \
    mv bin/consul /bin && \
    rm -rf $GOPATH && \
    apk del go git gcc musl-dev make bash && \
    rm -rf /var/cache/apk/* && \
    addgroup consul && \
    adduser -D -G consul consul && \
    mkdir -p /data/consul && \
    chown -R consul:consul /data/consul
```
Putting aside @tgross 's hate for go get. This would give you a version of Consul that was compiled for the platform.  For some reason, I trust a tagged Hashicorp release from github more than I do an entire glibc that I've curled into my environment. In fact, we should probably just create an APK for it and submit it to Alpine.